libs/coin-evm/src/__tests__/fixtures/synchronization.fixtures.ts

```diff
@@ -20,17 +21,24 @@ export const currency: CryptoCurrency = Object.freeze({
   ...getCryptoCurrencyById("ethereum"),
   ethereumLikeInfo: {
     chainId: 1,
-    node: {
-      type: "external" as const,
-      uri: "https://my-rpc.com",
-    },
-    explorer: {
-      type: "etherscan" as const,
-      uri: "https://api.com",
-    },
   },
 });
 
+setCoinConfig((): any => {
```

Comment:
I'm scared this edits the coinConfig globally, Ideally we want to mock the returned value of getConfig call we can do this together :)

another dirty solution is to store in beforeAll and reapply in afterAll but not clean

To apply on every tests

Reply:
In the tests we usually have a `beforeEach` that overrides this one. but it's still needed before in the file we call `getSyncHash` that requires a coinConfig set up

Reply:
I tend to agree with Hedi here.
If you don't want to use a mock, call a `afterAll` to erase the CoinConfig setted this way. It will avoid side effects.